src/main/java/com/google/edith/GroceryNameProcessor.java

```diff
@@ -0,0 +1,42 @@
+package com.google.edith;
+
+import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
+import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.EncodingType;
+import com.google.cloud.language.v1.Entity;
+import com.google.cloud.language.v1.EntityMention;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class GroceryNameProcessor {
+  public String process(String text) throws Exception {
+    try (LanguageServiceClient language = LanguageServiceClient.create()) {
```

Comment:
This line is causing the test failure. This should be mocked.

```java
public final class GroceryNameProcessor {
  
  private final LanguageServiceClient client;

  public GroceryNameProcessor() {
    this.client = LanguageServiceClient.create();
  }

  GroceryNameProcessor(LanguageServiceClient client) {
    this.client = client;
  }
}
```

If you need to create these on a per-request basis you can use a Provider

```java

public final class GroceryNameProcessor {

  public interface LanguageServiceClientProvider {
    LanguageServiceClient get(); // possibly add "throws Exception" if necessary
  }
  
  private final LanguageServiceClientProvider clientProvider;

  public GroceryNameProcessor() {
    this.clientProvider = () -> LanguageServiceClient.create();
  }

  GroceryNameProcessor(LanguageServiceClientProvider clientProvider) {
    this.clientProvider = clientProvider;
  }

  public String process(String text) throws Exception {
    try (LanguageServiceClient language = clientProvider.get()) {
      ...
    }
  }
}
```